perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -49,7 +49,7 @@ git fetch origin -q
 
 function execute_perf_test() {
   mkdir -p gcs
-  GCSFUSE_FLAGS="--implicit-dirs --max-conns-per-host 100"
+  GCSFUSE_FLAGS="--implicit-dirs"
```

Comment:
please also change other CI scripts like perfmetrics/scripts/ls_metrics/listing_benchmark.py & tf and pytorch long running tests.

Reply:
Done